content/en/database_monitoring/setup_mysql/rds.md

```diff
@@ -40,6 +41,10 @@ Proxies, load balancers, and connection poolers
 Data security considerations
 : See [Sensitive information][2] for information about what data the Agent collects from your databases and how to ensure it is secure.
 
+## Configure the AWS integration
+
+Enable Standard Collection in the Resource Collection section of your Amazon Web Services integration tile.
```

Comment:
Would I bold "Standard Collection" and "Resource Collection" because they represent UI text?

Reply:
I would also recommend adding https://app.datadoghq.com/integrations/amazon-web-services as a second link!